draft-ietf-quic-transport.md

```diff
@@ -571,106 +630,262 @@ QUIC's connection establishment begins with version negotiation, since all
-  flag set and will include the server's set of supported versions.  On
-  subsequently received packets for the same connection ID with the unacceptable
-  version, the server MUST continue responding with a Version Negotiation
-  packet.
+If the packet contains a version that is acceptable to the server, the server
+proceeds with the handshake ({{handshake}}).  All subsequent packets sent by the
+server MUST have the VERSION flag unset.  This commits the server to the version
+that the client selected.
 
 When the client receives a Version Negotiation packet from the server, it should
 select an acceptable protocol version.  If such a version is found, the client
 MUST resend all packets using the new version, and the resent packets MUST use
 new packet numbers.  These packets MUST continue to have the VERSION flag set
 and MUST include the new negotiated protocol version.
 
-The client MUST send its version on all packets until it receives a packet from
-the server with the VERSION flag off.  If version negotiation is successful, the
-client should receive a packet from the server with the VERSION flag off
-indicating the end of version negotiation.  All subsequent packets the client
-sends MUST have the version flag off.
+The client MUST include its selected version on all packets until it starts
+protecting packets with 1-RTT keys.  Only unprotected packets and 0-RTT
+protected packets can include a version.  A client MUST NOT change the version
+it uses unless it is in response to a version negotiation packet from the
+server.
 
-Once the server receives a packet from the client with the VERSION flag off, it
-MUST ignore the VERSION flag in subsequently received packets.
+Version negotiation uses unprotected data.  The result of the negotiation MUST
+be revalidated once the cryptographic handshake has completed (see
+{{version-validation}}).
 
-The Version Negotiation packet is unencrypted and exchanged without
-authentication.  To avoid a downgrade attack, the client needs to verify its
-record of the server's version list in the Version Negotiation packet and the
-server needs to verify its record of the client's originally proposed version.
-Therefore, the client and server MUST include this information later in their
-corresponding crypto handshake data.
 
 ## Crypto and Transport Handshake {#handshake}
 
 QUIC relies on a combined crypto and transport handshake to minimize connection
-establishment latency.  QUIC provides a dedicated stream (Stream ID 1) to be
-used for performing a combined connection and security handshake (streams are
-described in detail in {{streams}}).  The crypto handshake protocol encapsulates
-and delivers QUIC's transport handshake to the peer on the crypto stream.  The
-first QUIC packet from the client to the server MUST carry handshake information
-as data on Stream ID 1.
+establishment latency.  QUIC allocates stream 1 for the cryptographic handshake,
+which uses TLS.
```

Comment:
What is that value?  Because all I'm seeing right now is a section that is badly out of date and in need of maintenance.  The easiest maintenance would be deleting it.

I will open a thread on the list.